com.yktoo.ymuse.yml

```diff
@@ -0,0 +1,108 @@
+app-id: com.yktoo.ymuse
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: ymuse
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+rename-icon: ymuse
+rename-desktop-file: ymuse.desktop
+modules:
+  - name: ymuse
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOPATH: /run/build/ymuse/.go
+    build-commands:
+      # Remove versions from the downloaded sources
+      # (e.g. vendor/github.com/pkg/errors@v0.9.1 -> vendor/github.com/pkg/errors)
+      - find vendor/ -type d -name '*@v*' |
+        while read d; do mv "$d" "$(echo "$d" | sed -E 's/@v.*//')"; done
+
+      # Write out a vendor modules file. Generate with "go mod vendor" (in ymuse dir) and collect from vendor/
+      - |
+        cat <<EOF > vendor/modules.txt
```

Comment:
not a blocker, but my recommendation here is to make this a file as well. It will be way easier when you or if you need to regenerate it than editing the yaml.

(it's also good practice)

Reply:
Agreed